libs/components/list-builder-view-checklist/src/lib/modules/list-view-checklist/list-view-checklist.component.html

```diff
@@ -1,56 +1,57 @@
-<div class="sky-list-view-checklist" *ngIf="active | async">
-  <div [ngSwitch]="selectMode">
-    <ng-template ngSwitchCase="single">
-      <div role="radiogroup">
+<div *ngIf="active | async" class="sky-list-view-checklist">
+  <div>
```

Comment:
is this div necessary? I can see the reasoning to keep it to keep the DOM consistent, but this makes it just a extraneous element right?

Reply:
Removed